src/keys.ts

```diff
@@ -1,6 +1,7 @@
 export const isDarwin = /Mac|iPod|iPhone|iPad/.test(window.navigator.platform);
 export const isWindows = /^Win/.test(window.navigator.platform);
 export const isAndroid = /\b(android)\b/i.test(navigator.userAgent);
+export const isIPad = true; ///iPad/.test(window.navigator.platform);
```

Comment:
why setting it to true ?

Reply:
See my comment above. I set this to true because I had a hunch that it does not resolve correctly for iOS15.1 (and my hunch proved to be correct. Setting this to fixed true the links started to work on my iOS15.1 iPad as well.